README.md

```diff
@@ -11,15 +11,41 @@ bundle
 
 to install all further dependencies.
 
-Run `bundle exec jekyll build`. This will create a directory `_site`.  To build
-for production environment set the environmental variable
-`JEKYLL_ENV=production`.  To watch for file changes and trigger a rebuild, you
-can run `bundle exec jekyll build --watch`.
+Run `make build`. This will create a directory `_site`. To watch
+for file changes and trigger a rebuild, you can set the environment variable
+`WATCH`:
 
-For local environments you need to add the `_config_development.yml` file to the
-build to override the `baseurl`, by adding `--config _config.yml,_config_development.yml` to
-the build command.
+`WATCH=1 make build`.
+
+To build for production environment set the environmental variable
+`PRODUCTION=1`.

```

Comment:
: instead of . (to be consistent with previous paragraph).

would also be good to use a consistent layout, compare, either use "```" (see bundle) or "`".

Reply:
done